### DISCUSSION TITLE
webview onPermissionRequest Issue

### DISCUSSION THREAD

**Discussion by ncrehman** [react-native-webview/react-native-webview#2009] [Q&A]
Hi All,
I am using webview in multiple project but I am getting error in one of my project after updating  "react-native-webview": "^11.6.1" as:
 error: method onPermissionRequest(android.webkit.PermissionRequest) is already defined in class com.reactnativecommunity.webview.RNCWebViewManager.RNCWebChromeClient
    public void onPermissionRequest(final PermissionRequest request) {

**Comment by armandchbr** [comment]
> Hi All,
> I am using webview in multiple project but I am getting error in one of my project after updating "react-native-webview": "^11.6.1" as:
> error: method onPermissionRequest(android.webkit.PermissionRequest) is already defined in class com.reactnativecommunity.webview.RNCWebViewManager.RNCWebChromeClient
> public void onPermissionRequest(final PermissionRequest request) {

Same here using the same version 11.6.1:
react-native-webview/android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java:1214: error: method onPermissionRequest(PermissionRequest) is already defined in class RNCWebChromeClient
    public void onPermissionRequest(final PermissionRequest request) {
